classes/webservice.php

```diff
@@ -384,6 +384,24 @@ public function get_user($identifier) {
 
         return $founduser;
     }
+    
+    /**
+     * @param string $identifier The user's email or the user's ID per Zoom API.
+     * @return array|false If schedulers are returned array of {id,email} objects. Otherwise returns false.
+     * @link https://marketplace.zoom.us/docs/api-reference/zoom-api/users/userschedulers
+     */
+    public function get_schedule_for_users($identifier) {
+        $url = "users/{$identifier}/schedulers";
+        
+        $schedulers = false;
```

Comment:
Not sure which style @rlorenzo prefers, but it might be nice if this always returned an array.
```suggestion
        $schedulers = [];
```

Really, since this value is just the default, you can probably move it into the `catch`